ops/ansible/playbooks-ccs/roles/platinum_post/tasks/main.yml

```diff
@@ -0,0 +1,14 @@
+## Final tasks to be run when building the platinum image.
+
+## AIDE fixup
+# remove the failing aide cron job
+- name: 'Ensure failing aide cron job is removed.'
+  become: true
+  ansible.builtin.cron:
```

Comment:
Did BFD upgrade Ansible everywhere?  I see the new full path task definitions, looks like those are only compatible with Ansible >= 2.10.  Just checking!

Reply:
I assumed we were at least to 2.10, but could be mistaken. I'll refactor this just in case.